docs/wp-scripts.md

```diff
@@ -0,0 +1,53 @@
+# Reusable workflows – WordPress Scripts
+
+## Lint
+
+This reusable workflow contains internally support for linting "styles" (`stylelint`), "scripts" (`eslint`) and markdown files (`markdownlint`) by executing the `@wordpress/scripts` executable `wp-scripts`.
+
+**Simplest possible example:**
+
+```yml
+name: Front-Office QA
+on: [push]
+jobs:
+  wp-scripts-lint:
+    uses: inpsyde/reusable-workflows/.github/workflows/wp-scripts-lint.yml@main
+```
+
+### Configuration parameters
+
+#### Inputs
+
+| Name                  | Default                                       | Description                                                                                  |
+|-----------------------|-----------------------------------------------|----------------------------------------------------------------------------------------------|
+| `NPM_REGISTRY_DOMAIN` | `https://npm.pkg.github.com/`                 | Domain of the private npm registry                                                           |
+| `NODE_VERSION`        | 16                                            | Node version with which the assets will be compiled                                          |
+| `PACKAGE_MANAGER`     | `yarn`                                        | Package manager with which the dependencies should be installed (`npm` or `yarn`)            |
+| `NODE_OPTIONS`        | `''`                                          | Space-separated list of command-line Node options                                            |
+| `LINT_TOOLS`          | `'["js", "style", "md-docs"]'`                | An array of tools supported by wp-scripts lint-*. Supported are "js", "style" and "md-docs". |
+| `ESLINT_ARGS`         | `'--format stylish'`                          | Set of arguments passed to wp-script lint-js.                                                |
+| `STYLELINT_ARGS`      | `'--formatter github'`                        | Set of arguments passed to wp-script lint-style.                                             |
```

Comment:
Please remove the full stops (`.`) from the descriptions and reformat the table.

Reply:
Removed here: https://github.com/inpsyde/reusable-workflows/pull/49/commits/7e049e35e9fd08d4d8c7fa027745c2d08ba27f94